diode/sdfv.py

```diff
@@ -11,6 +12,49 @@
 import jinja2
 
 
+def view(sdfg, filename=None):
```

Comment:
This is just a change from my autodiff branch: this allows you to view sdfgs from the terminal;
```python
from diode.sdfv import view
view(sdfg) # this will write the html to a tempfile and open it in your browser
```

Reply:
Nice